Command/GenerateDtoCommand.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace WsSys\DtoGeneratorBundle\Command;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console;
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+use Sensio\Bundle\GeneratorBundle\Command\GeneratorCommand;
+use WsSys\DtoGeneratorBundle\Generator;
+
+/**
+ * Command to generate the Dtos
+ */
+class GenerateDtoCommand extends GeneratorCommand
+{
+    /**
+     * All the supported Types eg: xsd, json
+     * @var array 
+     */
+    protected $supportedTypes = array('xsd', 'json');
+    
+    /**
+     * Type of the source
+     * 
+     * @var string 
+     */
+    protected $type;
+    
+    /**
+     * @see Console\Command\Command
+     */
+    protected function configure()
+    {
+    	$this
+        ->setName('ws:generator:generate:dto')
+        ->setAliases(array('ws:generator:dto:generate'))
+        ->setDescription('Convert Xml/Json into PHP.')
+        ->setDefinition(array(
+            new InputArgument(
+                'src', InputArgument::REQUIRED, 'The path of the souce XSD or JSON.'
+            ),
+            new InputArgument(
+                'destination', InputArgument::REQUIRED, 'The path to save DTOs.'
+            ),
+            new InputArgument(
+                'destination-namespace', InputArgument::REQUIRED, 'The target namespace for DTOs'
+            ),
+            new InputArgument(
+                'src-type', InputArgument::REQUIRED, 'The type of the source eg: XSD'
```

Comment:
I would put ```implode(', ', $this->supportedTypes)``` at the end of the description to help the user in knowing what to input in there.